core/control-plane/control-plane-transfer/src/main/java/org/eclipse/edc/connector/controlplane/transfer/flow/TransferTypeParserImpl.java

```diff
@@ -23,12 +23,19 @@
 
 public class TransferTypeParserImpl implements TransferTypeParser {
 
+    /**
+     * Parses a compose transfer type string into a {@link TransferType}:
+     * {@code DESTTYPE-{PUSH|PULL}(-RESPONSETYPE)}, for example {@code HttpData-PULL/Websocket}
+     *
+     * @param transferType the transfer type string representation.
+     * @return a {@link TransferType}
+     */
     @Override
     public Result<TransferType> parse(String transferType) {
         Optional<Result<TransferType>> parsed = Optional.ofNullable(transferType)
                 .map(type -> type.split("-"))
```

Comment:
shouldn't it be parsed by `/` and then by `-`?

Reply:
@wolf4ood was having doubts about the "/" causing troubles with json-ld